.github/workflows/dis-apim-test.yml

```
@@ -0,0 +1,36 @@
+name: Dis-apim-operator Tests
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - services/dis-apim-operator/**
+      - .github/workflows/dis-apim-test.yml
+  pull_request:
+    branches:
+      - main
+    paths:
+      - services/dis-apim-operator/**
+      - .github/workflows/dis-apim-test.yml
+
+jobs:
+  test:
+    name: Run on Ubuntu
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: services/dis-apim-operator
+    steps:
+      - name: Clone the code
+        uses: actions/checkout@v4
+
+      - name: Setup Go
+        uses: actions/setup-go@v5
+        with:
```

Can also be like this to avoid changing the go version:
```suggestion
      with:
          go-version-file: '${{ env.WORKDIR }}/go.mod'
          cache: true
```